ncm-spma/src/main/perl/spma/yum.pm

```diff
@@ -1015,7 +1015,7 @@ sub configure_yum
         $fh->add_or_replace_lines(
             $name,
             $name. q{\s*=\s*}.$valuereg.'$',
-            "\n$name=$value\n",
+            "$name=$value\n",
```

Comment:
what if the file doesn't end with a newline? i'm not sure FileEditor tests this...

Reply:
seems it does by default: https://github.com/quattor/CAF/blob/master/src/main/perl/FileEditor.pm#L299 and https://github.com/quattor/CAF/blob/master/src/main/perl/FileEditor.pm#L349

can you modify the unittest below to check this?

Reply:
FileEditor should deal with that, not the components.

Reply:
The match should still work though...

Reply:
Done, the noaction test now starts with a file that has a missing newline at EOF.